www/src/layouts/index.js

```diff
@@ -13,13 +13,13 @@
 
 import React, {Component} from 'react';
 import docsearch from 'docsearch.js';
-import Flex from 'components/Flex';
-import Footer from './components/Footer';
-import Header from './components/Header';
+import Flex from '../components/Flex';
+import Footer from '../layout-components/components/Footer';
```

Comment:
Hm. Yeah. I initially started out with everything in a single `/components` directory but moved away from it b'c it felt too spread out.